unattended_scripts/install_functions/opendistro/elasticsearch.sh

```diff
@@ -7,7 +7,7 @@
 
 installElasticsearch() {
 
-    logger "Installing Open Distro for Elasticsearch."
+    logger "Starting the Elasticsearch Installation"
```

Comment:
Same here.
This should be so? or are we already changing the name to Wazuh indexer??